source/class/qx/ui/table/cellrenderer/Abstract.js

```diff
@@ -168,7 +168,7 @@ qx.Class.define("qx.ui.table.cellrenderer.Abstract",
      */
     _getCellAttributes : function(cellInfo)
     {
-      return "";
+      return `role=gridcell aria-readonly=${!cellInfo.editable}`;
```

Comment:
If this is to be backported to v6, which supports the legacy python generator, the backtick syntax probably won't work. Can you use ES5 syntax here, just to be sure?